python/setup.cfg

```diff
@@ -2,20 +2,9 @@
 tag_build =
 
 [tool:pytest]
-timeout = 100
-timeout_method = thread
-addopts = --random --tb=short --cov pyhive --cov-report html --cov-report term --flake8
+addopts = -n 4 --randomly-seed=992992726 --tb=short --cov pyhive --cov-report html --cov-report term
```

Comment:
why `--randomly-seed` is required?

Reply:
trying to replicate the existing setup, can be removed